src/bin/cql-stress-scylla-bench/stats.rs

```diff
@@ -252,7 +252,12 @@ impl StatsPrinter {
         Ok(())
     }
 
-    pub fn print_final(&self, stats: &Stats, out: &mut impl Write) -> Result<()> {
+    pub fn print_final(
+        &self,
+        stats: &Stats,
+        errors: &[anyhow::Error],
+        out: &mut impl Write,
+    ) -> Result<()> {
```

Comment:
There will be at most as many error as the `concurrency` parameter. So, in the worst case, it will be thousands.

Besides, this is what scylla-bench does, and I'm striving for compatibility - maybe not with regards to exact error strings, but formatting at least...